scene/main/viewport.cpp

```diff
@@ -1632,7 +1632,9 @@ void Viewport::_gui_call_notification(Control *p_control, int p_what) {
 
 Control *Viewport::gui_find_control(const Point2 &p_global) {
 	ERR_MAIN_THREAD_GUARD_V(nullptr);
-	// Handle subwindows.
+	// `gui_find_control` doesn't take embedded windows into account. So the caller of this function
```

Comment:
I would put this comment above the function so that certain editors/IDEs show it in the info popup, e.g. like in VSCode:

![image](https://github.com/user-attachments/assets/b0dc22d8-5524-4652-8dd5-69227441b477)